### ISSUE TITLE
Add goconvey support

### ISSUE THREAD

**Issue by zolotov** [go-lang-plugin-org/go-lang-idea-plugin#1976]

**Comment by dlsniper** [comment]
Should one use as a reference implementation the current implementation for gocheck (and the PR from https://github.com/go-lang-plugin-org/go-lang-idea-plugin/pull/1673 or that PR is too complex as it was also changing the internals a bit?)

**Comment by zolotov** [comment]
@dlsniper I don't think that it should be referenced to that PR, I've reworked test frameworks support after that. For adding new framework it's enough to add new implementation of GoTestFramework, GoTestRunConfigurationProducerBase and editing GoTestRunConfigurationEditorForm.
